doc/release_notes/release_notes_2.4.rst

```diff
@@ -38,9 +38,160 @@ What's New in v2.4
 ACRN Configuration and Build
 ============================
 
-[Add a short description of the changes to the ACRN configuration and build
-process that no longer uses Kconfig and relies on XML files, and refer them to
-the updated building from source and configuration options documents]
+The following major changes on ACRN configuration and build process have been
+integrated into v2.4:
+
+ - Metadata of configuration entries, including documentation and attributes,
+   has been removed from ``scenario`` XMLs.
+ - The C sources generated from ``board`` and ``scenario`` XMLs are no longer
+   maintained in the repository. Instead they'll be generated as part of the
+   hypervisor build. Users can now find them under ``configs/`` of the build
+   directory.
+ - Kconfig is no longer used for configuration. Related build targets, such as
+   ``defconfig``, now apply to the configuration files in XML.
+ - The ``make`` command-line variables ``BOARD`` and ``BOARD_FILE`` have been
+   unified. Users can now specify ``BOARD=xxx`` when invoking ``make`` with
+   ``xxx`` being either a board name or a (relative or absolute) path to a
+   board XML. ``SCENARIO`` and ``SCENARIO_FILE`` have been unified in the same
+   way.
+
+For complete instructions to get started with the new build system, refer to
+:ref:`getting-started-building`. For an introduction on the concepts and
+workflow of the new configuration mechanism, refer to
+:ref:`acrn_configuration_tool`.
+
+Upgrading to v2.4 From Previous Releases
+****************************************
+
+Users are highly recommended to follow the instructions below in order to
+upgrade to v2.4 from previous ACRN releases.
+
+Additional Dependencies
+=======================
+
+Python 3.6 or above is required to build ACRN v2.4. If you are using an old
+version of Ubuntu (e.g. 16.04), you can use PPA (e.g. `New Python Versions
+<https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa>`_ from deadsnakes
+team) for recent Python versions.
+
+In addition, the following tools and packages are needed to build ACRN:
```

Comment:
Can we make it clear that this is not the complete list but rather new tools that need to be installed compared to v2.3

I also ran into an issue whereby I had to install `libxml2-utils` which is currently not mentioned in our documentation (see PR #5862). This should also be mentioned here.

Reply:
Thanks for pointing this out. I've added a note to call out that this is not the complete list. Also added libxml2-utils here.